tests/onnxruntime/test_modeling.py

```diff
@@ -3920,8 +3920,7 @@ def test_compare_generation_to_io_binding(
 
 
 class ORTModelForVision2SeqIntegrationTest(ORTModelTestMixin):
```

Comment:
There seem to be no test for logits match with/without IO Binding in this class, can you add it?

Reply:
done! but I had to add a model type `donut` to tasks manager, even though it's not exactly a model type. is that okay in this case ?

Reply:
I guess tests are passing now (on linux). I had some problems with `ORT_CUDA_UNAVAILABLE`. I can perfectly run models with `CUDAExeutionProvider` but the pipelines provider check fails because of `ORT_CUDA_UNAVAILABLE`.

Reply:
Are you sure the model is indeed dispatched to CUDA EP? You can check with

```python
session_options = onnxruntime.SessionOptions()
session_options.log_severity_level = 0

ort_model = ORTModel.from_pretrained(
    ...,
    session_options=session_options
)
```